performance-tests/commoncrawl-project/converted-entities/Location.entity.json

```diff
@@ -0,0 +1,43 @@
+{
+  "info" : {
+    "title" : "Location",
+    "version" : "0.0.1",
+    "baseUri" : "http://marklogic.com/cc/"
+  },
+  "definitions" : {
+    "Location" : {
+      "required" : [ ],
+      "pii" : [ ],
+      "elementRangeIndex" : [ "ip_from", "ip_to", "country_name", "region_name", "city" ],
```

Comment:
Instead of using elementRangeIndex or rangeIndex, add "facetable": true to these property definitions.

Reply:
Oops, just realized this is the converted-entities directory - you can safely delete this directory